### PR TITLE
Add One  keyboard's sounds

### PR DESCRIPTION
**Pull request by MysteriousLog6**
Added one  keyboard's sound , a bit busy due to exams but will be adding more over the coming weeks

### CODE REVIEW

File: src/js/sound.js [hunk 1]
```diff
@@ -115,6 +115,28 @@ export function init() {
...
+        ],
+        counter: 0,
+      },
+           {
+        sounds: [
+          new Audio("../sound/click5/click5_5.wav"),
+          new Audio("../sound/click5/click5_6.wav"),
+        ],
+        counter: 0,
+      },
```
Comment (Miodec) [inline src/js/sound.js:139]: These sounds are in the `4` property of sounds. It needs to be in its own property, same as its id (in this case `5`)

**Review by Miodec — CHANGES_REQUESTED** [review]

File: src/js/sound.js (hunk 1 above)
Comment (MysteriousLog6) [reply src/js/sound.js:139]: Sorry about that will fix it!

Comment (MysteriousLog6) [reply src/js/sound.js:139]: I have fixed it in the main branch of mine

**Comment by MysteriousLog6** [comment]
I have fixed it

File: src/js/commandline.js [hunk 2]
```diff
@@ -852,6 +852,19 @@ let commandsSoundOnClick = {
...
+      id: "setSoundOnClick4",
+      display: "5",
+      exec: () => {
+        setPlaySoundOnClick("5");
+        Sound.playClick(config.playSoundOnClick);
+          {
+      id: "setSoundOnClick5",
+      display: "5",
+      exec: () => {
+        setPlaySoundOnClick("5");
```
Comment (Miodec) [inline src/js/commandline.js:867]: Duplicate entry, also it seems like the curly brackets are messed up here

**Comment by Miodec** [comment]
Also you didnt seem to fix the previous issue (or didn't push it to the remote repo)

**Comment by MysteriousLog6** [comment]
I am new to doing pull requests so I really don't know how to allow the changes requested.

**Comment by Miodec** [comment]
> I am new to doing pull requests so I really don't know how to allow the changes requested.

You don't just approve them, you need to make the changes yourself and push them to your remote master branch.

**Comment by MysteriousLog6** [comment]
Well I did do that will try doing it later , windows is updating so can't do anything on my laptop.

**Comment by UnrealApex** [comment]
@MysteriousLog6 

### Documentation on Pull Requests 
----
- [About Pull Requests](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
- [Basic Overview](https://guides.github.com/activities/hello-world/)
- [Creating a Pull Request](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request)
- [Github Learning Lab](https://lab.github.com/)

Hope this helps.

**Comment by Miodec** [comment]
FYI there are a lot of commits unrelated to the original pull request so I cant really merge this.

**Comment by MysteriousLog6** [comment]
I will have to come back to this once my exam is gone.

**Comment by MysteriousLog6** [comment]
Removing pull request will redo after a few days (A bit busy currently).

**Pull request closed without merging**